src/sdk/namespace_fs.js

```diff
@@ -2921,13 +2921,13 @@ class NamespaceFS {
                 const versioned_path = latest_ver_info && this._get_version_path(params.key, latest_ver_info.version_id_str);
                 const versioned_info = latest_ver_info && await this._get_version_info(fs_context, versioned_path);
 
-                gpfs_options = is_gpfs ?
+                dbg.log1('Namespace_fs._delete_latest_version:', latest_ver_info, versioned_path, versioned_info, gpfs_options);
```

Comment:
I think you can remove the `gpfs_options` from this printing since it will stay with an `undefined` value.
```suggestion
                dbg.log1('Namespace_fs._delete_latest_version:', latest_ver_info, versioned_path, versioned_info);
```